application.py

```diff
@@ -9,11 +9,13 @@
 def index():
     if request.method == 'POST':
         file = request.files['file']
-
+        if file.filename == 'blob':
+            file.filename = 'blob.png'
```

Comment:
Why are we doing this here?

Reply:
When we are doing `fd.append("file", blob)` , I could not really name it with an extension `.png` so I have to manually add it then because of the checks we are doing. Was not sure what would be the most clean/safe approach for this.

Reply:
This was just a hack when you were saving files locally. We dont do that anymore so I removed it.